v0/destinations/marketo/util.js

```diff
@@ -122,7 +122,7 @@ const sendGetRequest = async (url, options) => {
  * @returns { response, status }
  */
 const sendPostRequest = async (url, data, options) => {
-  const clientResponse = await httpPOST(url, options);
+  const clientResponse = await httpPOST(url, data, options);
```

Comment:
How it was working previously as we are sending options in place of data?

Reply:
Thats what perplexing, no-one used this path